system-parachains/asset-hub-paseo/src/lib.rs

```diff
@@ -1099,6 +1119,43 @@ impl pallet_asset_conversion::Config for Runtime {
 	>;
 }
 
+parameter_types! {
+	pub const DepositPerItem: Balance = system_para_deposit(1, 0);
+	pub const DepositPerByte: Balance = system_para_deposit(0, 1);
+	pub CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(30);
+}
+
+impl pallet_revive::Config for Runtime {
+	type Time = Timestamp;
+	type Currency = Balances;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type DepositPerItem = DepositPerItem;
+	type DepositPerByte = DepositPerByte;
+	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
+	// TODO(#840): use `weights::pallet_revive::WeightInfo` here
```

Comment:
We are good using the weights shipped with the pallet for now. That way costs resemble closely what users might see in prod.